docs/content/install.sh

```diff
@@ -9,9 +9,23 @@ if [ -n "$1" ] && [ "$1" != "beta" ]; then
 fi
 
 if [ -n "$1" ]; then
-    install_beta="yes"
+    install_beta="beta "
 fi
 
+#check installed version of rclone to determine if update is necessary
+version=`rclone --version | head -n 1`
+if [ -z "${install_beta}" ]; then
+    current_version=`curl https://downloads.rclone.org/version.txt`
+else
+    current_version=`curl https://beta.rclone.org/version.txt`
+fi
+
+if [ "$version" = "$current_version" ]; then
+    echo && echo "The latest ${install_beta}version of rclone is already installed" && echo
```

Comment:
Can you put the version in this message too? I think that would be helpful.

"The latest ${install_beta}version of rclone ${version} is already installed"

Reply:
Sure will do.